NppJSONViewer/NppJsonViewer/resource.h

```diff
@@ -40,6 +40,7 @@
 #define IDC_CHK_FORMAT_ON_OPEN          1029
 #define IDC_CHK_IGNORE_COMMA            1030
 #define IDC_CHK_IGNORE_COMMENT          1031
+#define IDC_CHK_REPLACE_UNDEFINED       1032
```

Comment:
Increase to `1033`
`#define _APS_NEXT_CONTROL_VALUE 1032`

Reply:
Oops!